internal/runners/prepare/prepare.go

```diff
@@ -38,19 +37,20 @@ func New(prime primeable) *Prepare {
 func (r *Prepare) Run() error {
 	logging.Debug("ExecutePrepare")
 
-	binDir := filepath.Join(config.CachePath(), "prepareBin")
+	binDir := filepath.Join(config.CachePath(), "bin")
 	fail := fileutils.Mkdir(binDir)
 	if fail != nil {
-		return locale.WrapError(fail.ToError(), "err_prepare_bin_dir", "Could not create global installations directory")
+		return locale.WrapError(fail.ToError(), "err_prepare_bin_dir", "Could not create bin directory")
 	}
 
 	envUpdates := map[string]string{
-		"PATH": fmt.Sprintf("%s%s%s", binDir, string(os.PathListSeparator), os.Getenv("PATH")),
+		"PATH": binDir,
 	}
 
-	fail = r.subshell.WriteUserEnv(envUpdates, true)
+	fail = r.subshell.WriteUserEnv(envUpdates, sscommon.Default, true)
 	if fail != nil {
-		return locale.WrapError(fail.ToError(), "err_prepare_update_env", "Could not update user environment")
+		logging.Debug("Encountered failure attempting to update user environment: %s", fail.ToError())
+		r.out.Notice("Could not update user environment")
```

Comment:
This should still fail on Mac and Windows. My review said "on Linux".

Reply:
Whoops, missed that.